package-lock.json

```diff
@@ -11,7 +11,6 @@
 			"dependencies": {
 				"@codemirror/lang-json": "^6.0.1",
 				"@uiw/react-codemirror": "^4.21.24",
-				"@wordpress/env": "^9.9.0",
```

Comment:
This package is installed as `devDependencies` in `package.json`. Therefore, it is unclear why it existed as `dependencies` in `package-lock.json`.